### PR TITLE
RewardsManager: rename lastInteractionBurnEpoch to lastClaimedEpoch

### PR DESCRIPTION
**Pull request by grandizzy**
- rename to last claimed epoch since all the interactions are claiming (update bucket exchange rate too)

### CODE REVIEW

File: src/RewardsManager.sol [hunk 1]
```diff
@@ -485,20 +485,20 @@ contract RewardsManager is IRewardsManager {
 
     /**
      *  @notice Retrieve an array of burn epochs from which a depositor has claimed rewards.
-     *  @param  lastInteractionBurnEpoch_ The last burn period in which a depositor interacted with the rewards contract.
-     *  @param  burnEpochToStartClaim_    The most recent burn period from a depostor earned rewards.
-     *  @return burnEpochsClaimed_        Array of burn epochs from which a depositor has claimed rewards.
+     *  @param  lastClaimedEpoch_      The last burn period in which a depositor interacted with the rewards contract.
```
Comment (ith-harvey) [inline src/RewardsManager.sol]: can we change the comment here to reflect that this was when the last "claim" occurred?

File: src/RewardsManager.sol [hunk 2]
```diff
@@ -255,7 +255,7 @@ contract RewardsManager is IRewardsManager {
         return (
             stakes[tokenId_].owner,
             stakes[tokenId_].ajnaPool,
-            stakes[tokenId_].lastInteractionBurnEpoch);
+            stakes[tokenId_].lastClaimedEpoch);
```
Comment (ith-harvey) [inline src/RewardsManager.sol]: Shouldn't this method have natspec?

Comment (grandizzy) [reply src/RewardsManager.sol]: It has inheritdoc here https://github.com/ajna-finance/contracts/blob/6b1133b6dd89a8a9cbd72e16f798e3898f908ed3/src/RewardsManager.sol#L251

File: src/RewardsManager.sol (hunk 1 above)
Comment (grandizzy) [reply src/RewardsManager.sol]: done in https://github.com/ajna-finance/contracts/pull/544/commits/d03af0a84daf16025888274b992ec0934ec99b61

**Review by ith-harvey — COMMENTED** [review]
Request this follows the template structure

**Review by MikeHathaway — APPROVED** [review]
LGTM

**Review by ith-harvey — APPROVED** [review]

**Pull request merged**